### PR TITLE
fix(versionInfo): add border to marker

### LINKED ISSUE #2669: Wrong style of unread marker
__Describe the Bug__

The unread marker on the version information in the status bar should have the same style as the unsaved marker on tabs. What is missing there is the dark blue border to increase the contrast of the dot.

Correct:

Screen Shot 2022-01-13 at 09 07 43

Missing border:

Screen Shot 2022-01-13 at 09 07 49

__Expected Style__

Same style for both markers. Ensure that the computed overall size of the unread marker remains 8x8px when adding the border (6x6px dot + 1px border).

### PR DESCRIPTION
**Pull request by MaxTru**
closes #2669
Screenshot_20220114_144332

### CODE REVIEW

File: client/src/plugins/version-info/VersionInfo.less
Comment (pinussilvestrus) [inline client/src/plugins/version-info/VersionInfo.less:10]: C[ontinuing a conversation I've had with @smbea recently](https://github.com/camunda/camunda-modeler/pull/2658#discussion_r781027716), can we promote the new way of using color variables in our components when possible? 

That would be cool 🙂

Comment (MaxTru) [reply client/src/plugins/version-info/VersionInfo.less]: Ups - sure thing!

Done, thanks!

**Review by pinussilvestrus — APPROVED** [review]
LGTM 👍

**Comment by andreasgeier** [comment]
@MaxTru pls provide an artifact to test

**Comment by MaxTru** [comment]
Artifacts will be available in 30-45mins via:
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/versionInfoCorrection/camunda-modeler-versionInfoCorrection-linux-x64.tar.gz
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/versionInfoCorrection/camunda-modeler-versionInfoCorrection-mac.dmg
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/versionInfoCorrection/camunda-modeler-versionInfoCorrection-mac.zip
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/versionInfoCorrection/camunda-modeler-versionInfoCorrection-win-ia32.zip
https://camunda-modeler-on-demand.s3.eu-central-1.amazonaws.com/versionInfoCorrection/camunda-modeler-versionInfoCorrection-win-x64.zip

Build link:
https://github.com/camunda/camunda-modeler/actions/runs/1707384039

**Review by andreasgeier — CHANGES_REQUESTED** [review]
Please remove the added box-shadow. Sorry if the request was misleading.
It's not needed here and it causes an unwanted glow on hover.

After removal, feel free to merge without another review.

**Comment by MaxTru** [comment]
> Please remove the added box-shadow. Sorry if the request was misleading. It's not needed here and it causes an unwanted glow on hover.
> 
> After removal, feel free to merge without another review.

Sure, thanks for review and the hint. Removed and will merge now.

**Pull request merged**
